src/IB/IBFEMethod.cpp

```diff
@@ -3662,6 +3662,12 @@ IBFEMethod::getFromInput(Pointer<Database> db, bool /*is_from_restart*/)
         d_default_interp_spec.use_nodal_quadrature = db->getBool("interp_use_nodal_quadrature");
     else if (db->isBool("IB_use_nodal_quadrature"))
         d_default_interp_spec.use_nodal_quadrature = db->getBool("IB_use_nodal_quadrature");
+    if (d_default_interp_spec.use_nodal_quadrature)
+    {
+        d_default_interp_spec.quad_type = QTRAP;
+        d_default_interp_spec.quad_order = FIRST;
+    }
```

Comment:
I'm not sure we have code in place to handle it, but we should disable adaptive quadrature here too.

Reply:
Yes. Also we should reorder the way things are setup to avoid overriding user settings. Will push an update in 1 minute.